### DISCUSSION TITLE
How to uninstall Auto Backup from HACS on CORE

### DISCUSSION THREAD

**Discussion by Domopanda** [jcwillox/hass-auto-backup#157] [Q&A] [answered]
Hello, I've tried to install this apparently useful Hach's integration on my CORE to mitigate the poor setting the default backup have, but unfortunately seems that AutoBackup doesn't work properly and I would delete it, because of clean list of just things working.
Btw i'm not able to remove as an integration while I was able to remove the entity....
Would you mind please, to explain how get rid of that in a clean way.. because now, without the entity I'm stuck in the middle of nowhere...aprreciate any helps...I'm sorry it looks like a nice integration if works :)

.thank's in advance :)

**Comment by steps56** [comment]
Just delete the integration from HACS.
Same way as you installed, always use HACS again and delete unused integration there.
Often its left stored at custom integratilns folder at HA.
Use a file explorer (I do it from Android Phone by "CX Explorer") to delete those unused imtergrations folder.
Alway make a backup of cpl. HA BEFORE doing.

**Comment by jcwillox** [comment, marked as answer]
Should be the same as uninstalling any integration from HACS, though you should probably delete the config entry first, that will cleanup the entity.
![image](https://github.com/user-attachments/assets/2bbbbba3-23e3-4e4e-9591-9daeea3ac79c)
You might need to reinstall the integration to do this.

Otherwise to truly nuke the installation, delete the `custom_components/auto_backup` folder, but this is what HACS will do for you when you uninstall it. Additionally, you can also remove the `.storage\auto_backup.snapshots_expiry` file, where it stores the expiry date of your backups.
